docs/releasing/publishing.md

```diff
@@ -48,32 +66,32 @@
   npm unlink ../govuk-frontend/package/
   ```
 
-11. Create a pull request and copy the changelog text.
+10. Create a pull request and copy the changelog text.
    When reviewing the PR, check that the version numbers have been updated and that the compiled assets use this version number.
 
-12. Once the pull request is approved, merge to **master**.
+11. Once the pull request is approved, merge to **master**.
 
-13. Checkout **master** and pull the latest changes.
+12. Checkout **master** and pull the latest changes.
 
-14. Log into npm (`npm login`), using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).
+13. Log into npm (`npm login`), using team [credentials](https://github.com/alphagov/design-system-team-credentials/tree/master/npm/govuk-patterns-and-tools).
 
-15. Run `npm run publish-release`, you will be prompted to continue or cancel.
+14. Run `npm run publish-release`, you will be prompted to continue or cancel.
 
-16. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases)
+15. View the created tag in the [Github interface](https://github.com/alphagov/govuk-frontend/releases)
   - select the latest tag
   - press 'Edit tag'
   - set "GOV.UK Frontend v[version-number]" as the title
   - add release notes from changelog
   - attach the generated ZIP that has been generated at the root of this project
   - publish release
 
-17. Log out from npm
+16. Log out from npm
 ```bash
 npm logout
 ```
 
-18. Once you've updated the GOV.UK Design System, Prototype Kit, and Frontend Docs, post a short summary of the release in the cross-government and GDS #govuk-design-system Slack channels. For example:
+## Steps you must complete after you release a new version of GOV.UK Frontend
 
-    🚀 We’ve just released GOV.‌UK Frontend v3.7.0. It's now easier and faster to use our Sass. We've also made improvements to back links, breadcrumbs, lists and the header. Thanks to @<SLACK-NAME> and @<SLACK-NAME> for helping with this release. [https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0](https://github.com/alphagov/govuk-frontend/releases/tag/v3.7.0)
+1. Update the GOV.UK Design System, Prototype Kit, and Frontend Docs (as in step 1 of [Steps you must complete before you release a new version of GOV.UK Frontend](#steps-you-must-complete-before-you-release-a-new-version-of-govuk-frontend)).
 
```

Comment:
Should there be something about posting the Slack summary we wrote in the earlier steps too?